### PR TITLE
Skip test_memory_exhaustion for Celestica E1031 and DX010

### PR DESCRIPTION
**Pull request by lizhijianrd**
### Description of PR

Summary:

As described in Azure/sonic-buildimage#10339, device `Celestica e1031` and `Celestica dx010` can't pass test case `platform_tests.test_memory_exhaustion`, we should temporarily skip this test case before the issue is fixed.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case improvement

### Back port request
- [ ] 201911
- [x] 202012

#### What is the motivation for this PR?

As descripted in Azure/sonic-buildimage#10339, test case `platform_tests.test_memory_exhaustion` causes device `Celestica e1031` and `Celestica dx010` offline from both ssh/console, so we should temporarily skip this test case before the issue is fixed.

#### How did you do it?

Add configurations in `tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml`.

#### How did you verify/test it?

Verified on physical DUTs.

#### Any platform specific information?

This PR only affects `Celestica e1031` and `Celestica dx010`

### CODE REVIEW

File: tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml [hunk 1]
```diff
@@ -550,6 +550,16 @@ platform_tests/fwutil/test_fwutil.py::test_fwutil_auto:
...
 
+#######################################
+#####  test_memory_exhaustion.py  #####
+#######################################
+platform_tests/test_memory_exhaustion.py:
+  skip:
+    reason: "Skip this test case until vendors fix the issue"
+    conditions:
+      - "hwsku in ['Celestica-DX010-C32', 'Celestica-E1031-T48S4']"
+      - https://github.com/Azure/sonic-buildimage/issues/10339
```
Comment (Blueve) [inline tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml:561]: Do we support put an external repo issue as condition here?

**Review by Blueve — CHANGES_REQUESTED** [review]

**Review by Blueve — APPROVED** [review]

File: tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml (hunk 1 above)
Comment (wangxin) [reply tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml:561]: Yes, the conditional mark plugin supports querying any Github issue.

**Review by wangxin — APPROVED** [review]

**Pull request merged**